src/django/api/views/v1/moderation_events.py

```diff
@@ -22,16 +24,31 @@
     import ModerationEventsQueryBuilder
 from api.views.v1.opensearch_query_builder.opensearch_query_director import \
     OpenSearchQueryDirector
+
 from api.views.v1.utils import (
     handle_errors_decorator,
-    serialize_params
+    serialize_params,
 )
 
 
 class ModerationEvents(ViewSet):
     swagger_schema = None
+
     permission_classes = [IsRegisteredAndConfirmed]
 
+    action_permissions = {
```

Comment:
I think it can be moved to the method level. I'm not sure we really need it at the class level since it's only used inside the `get_permissions` method.

Reply:
+1

Reply:
Good notice.
Moved it to the get_permissions method.
Thanks.